packages/flutter_test/lib/src/binding.dart

```diff
@@ -910,15 +910,14 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
         // Ideally, once the test has failed we would stop getting errors from the test.
         // However, if someone tries hard enough they could get in a state where this happens.
         // If we silently dropped these errors on the ground, nobody would ever know. So instead
-        // we report them to the console. They don't cause test failures, but hopefully someone
-        // will see them in the logs at some point.
```

Comment:
@Hixie You wrote this originally. Was there a reason for not just failing (as implemented in this PR)?